cmd/genji/shell/command.go

```diff
@@ -13,16 +13,34 @@ import (
 	"github.com/genjidb/genji/document"
 )
 
-var commands = []struct {
+type command struct {
 	Name        string
 	Options     string
+	DisplayName string
 	Description string
-}{
-	{".exit or exit", ``, "Exit this program."},
-	{".help or help", ``, "List all commands."},
-	{".tables", ``, "List names of tables."},
-	{".indexes", `[table_name]`, "Display all indexes or the indexes of the given table name."},
-	{".dump", `[table_name]`, "Dump database content or table content as SQL statements"},
+	Aliases     []string
+}
+
+var commands = []*command{
+	newCommandBuilder(".exit").
```

Comment:
This feels very Java-esque, I feel the same can be achieved by filling the struct directly:

```go
var commands = []*command{
  {
    name: ".exit",
    aliases: []string{"exit"},
    description: "Exit this program.",
  },
  {
    name: ".help",
    aliases: []string{"help"},
    description: "List all commands.",
  },
}

type command struct {
	Name        string
	Options     string
	DisplayName string
	Description string
}

func (c *command) String() string {
  ...
}
```